client/signup/index.js

```diff
@@ -19,6 +19,7 @@ module.exports = function() {
 		'/start/:flowName?/:stepName?/:stepSectionName?/:lang?',
 		adTracking.retarget,
 		controller.saveRefParameter,
+		controller.saveQueryObject,
```

Comment:
Given that the `ref` parameter is in the query object, do you think we could remove that? (Probably in a different PR).